### ISSUE TITLE
Not working in Ubuntu 20.10

### ISSUE THREAD

**Issue by kristoiv** [glouw/paperview#17]
Hello.

Just compiled it and ran it on a fresh Ubuntu 20.10 install. The program is running and I could confirm that it is running the rendering functions, but the wallpaper doesn't change from the system set one.

Is it not supported? Maybe Gnome gets in the way?

**Comment by glouw** [comment]
Yeah, I believe GNOME uses a compositor which paperview needs. Paperview works best with window managers that do not use compositors:

https://wiki.archlinux.org/index.php/window_manager

In the future I plan to write my own window manager with animated wallpapers built in which should hopefully help out a little more with the whole compatibility issue.

**Comment by DroWnThePoor** [comment]
@glouw 
I'm guessing this issue is still relevant and pertains to Ubuntu 18.04 with Gnome as well?
I just built paperview and converted a .gif, and I'm also seeing the process running with no change to background.
Perhaps the readme should mention Gnome isn't compatible with paperview.
I honestly forgot about Mutter being a compositor for Gnome, and some people may not even be aware of compositors in general.
